AleoFortune/Fortune
# Feature/package integration

## Description

Added and removed some packages to start building roulette game. 

yarn will be used instead of pnpmp so that cached pnpm locks also removed.


## Steps to test

- checkout the branch
- `yarn dev`
- check if is there any errors or not.
